packages/react-devtools-scheduling-profiler/src/constants.js

```diff
@@ -12,4 +12,4 @@ export {
   COMPACT_LINE_HEIGHT,
 } from 'react-devtools-shared/src/constants.js';
 
-export const REACT_TOTAL_NUM_LANES = 31;
+export const REACT_TOTAL_NUM_LANES = 30;
```

Comment:
This should always be 31. When we remove a lane, we re-allocate it to be either a TransitionLane or a RetryLane.